docs/reference/esql/esql-syntax.asciidoc

```diff
@@ -160,14 +160,17 @@ Datetime intervals and timespans can be expressed using timespan literals.
 Timespan literals are a combination of a number and a qualifier. These
 qualifiers are supported:
 
-* `millisecond`/`milliseconds`
-* `second`/`seconds`
-* `minute`/`minutes`
-* `hour`/`hours`
-* `day`/`days`
+* `nanosecond`/`nanoseconds`/`ns`
+* `microsecond`/`microseconds`/`us`
+* `millisecond`/`milliseconds`/`ms`
+* `second`/`seconds`/`s`
+* `minute`/`minutes`/`min`
+* `hour`/`hours`/`h`
+* `day`/`days`/`d`
 * `week`/`weeks`
```

Comment:
How about `w` as a shorthand for weeks? There doesn't seem to be any ambiguity (especially not around the near units).

Reply:
I've added a `q` for quarter too, btw.